lint-staged.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     // For that reason, we move the `--onlyChanged` flag next to it.
     'yarn lint:js --reporters=jest-silent-reporter --onlyChanged',
   ],
-  '*.{ts,tsx}': [
+  '!(cypress)/**/*.{ts,tsx}': [
```

Comment:
The Cypress `tsconfig.json` should be treated separately from the main one, therefore we need to differentiate here the checks